src/installation/guides/macbook.md

```diff
@@ -0,0 +1,53 @@
+# Macbookpro 12,1 (early 2015) 
+
+Everything is well supported on this laptop and install will work without problems. Post-installation, after having booted to your newly installed system, there is however a few points that need a little bit of work.
+
+## keymap are wrong
+This is a problem inherited by a few distros, including archlinux.
+tty keymaps for the macbooks (mac-fr, mac-us, mac-de) are completely broken and *you should not attempt to use them*, or it will result in an unusable keyboard and you will be forced to do a hard reset on your system. as a workaround generic keymaps
+work well (fr, us, de ..) you can use those for starter. later on you can find working ones and install/use them. here i've found a working one for the french macbook keyboards: [forums.archlinux.fr](https://forums.archlinux.fr/viewtopic.php?t=8679)
+
+```
+cd /usr/share/kbd/keymaps/mac/all
+wget ftp://ftp.linux-france.org/pub/macintosh/mac-fr-ext_new.map.gz
```

Comment:
If this is something we want to support, I think you need to submit package templates to provide Mac keymaps. We shouldn't have documentation asking users to fetch files from an arbitrary FTP server. Otherwise, this entire section can't be accepted.

Reply:
i'd be glad if you updated all these broken keymaps. this situation has been going on since 2015, at least on arch.

basicaly all mac-us, mac-fr, mac-de tty keymaps are broken and loading them make the keyboard unusable. hard-reset required.
by search and luck i did find this one i'm linking to. it works perfectly.

i have no experience in package making. i can learn and contribute. but don't wait for me, someone has to replace these broken keymaps before the new year 2016 ..oh wait it's 2019 now ;)